deployment/base/lexbox-deployment.yaml

```diff
@@ -53,23 +66,24 @@ spec:
       - name: lexbox-api
         # TODO: need to parameterize this tag
         image: ghcr.io/sillsdev/lexbox-api:develop
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
I had this set to Always so that a redeploy would fetch the latest version even if the image tag hasn't changed on staging. Can this go back?

Reply:
rolled back and applied to dev via patch